src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -97,37 +153,86 @@ private bool ApplicableOperatorIs(MethodSymbol candidate, CSharpSyntaxNode node,
             return true;
         }
 
-        private BoundPattern BindConstantPattern(
+        private BoundConstantPattern BindConstantPattern(
             ConstantPatternSyntax node,
             BoundExpression operand,
+            bool? operandIsNull,
```

Comment:
Currently it is done in one place up the call stack and passed down.

---
In reply to: [64946254](https://github.com/dotnet/roslyn/pull/11485#discussion_r64946254) [](ancestors = 64946254)